apps/wallet/src/ui/app/shared/text/index.tsx

```diff
@@ -58,6 +58,9 @@ const textStyles = cva([], {
             true: 'font-mono',
             false: 'font-sans',
         },
+        lineHeight: {
```

Comment:
You shouldn't need to do this, I think `p2` is the correct text config (we need to update our naming to align with the pbodysmall in figma)

Reply:
Ah ok thanks. I was trying to fit bodySmall so it was not working correctly

Reply:
Opened a PR to rename these classes to re-align with the figma: https://github.com/MystenLabs/sui/pull/11010